src/model/AppParameters.as

```diff
@@ -0,0 +1,5 @@
+package {
+public class AppParameters {
```

Comment:
removed. Had some ideas on this  file, but it looks that nothing to place there!